src/pages/enterprise/auth-connect.md

```diff
@@ -341,12 +360,53 @@ ___
 
 ▸ **onLogout**(): `void`
 
-Event handler which can be overriden to handle successful logout events
+Event handler which can be overridden to handle successful logout events
 
 **Returns:** `void`
 
 ___
 
+___
+<a id="ivconfig"></a>
+
+###  IVConfig
```

Comment:
Isn't a bunch of this Identity Vault related, not Auth Connect?

Reply:
update: "they are interfaces that we had to redefine to allow for AuthConnect to be used without IV. Rather than including them we have copies of them."

Reply:
@jcesarmobile FYI ^^^^